docs/SDK/StructuralAnalysisFormat.md

```diff
@@ -0,0 +1,108 @@
+# StructuralAnalysisFormat
+This package implements the [Structural Analysis Format](https://www.saf.guide) in C#
+
+The aim of this SDK is to allow anyone to rapidly get started implementing the SAF specification in their own software.
+
+## Features
+- Straight forward API (One import and one export service)
+- Validation
+- Compatibility support between versions
+- Effortlessly switch between metric & imperial values
+
+## Dependencies
+- Uses EPPlus (4.5.3.3) for handling excel files
+- Uses FluentValidation for validation
+- Uses UnitsNet for unit operations
+
+## What do you need to do?
+In order to use this SDK, it is sufficient for you to provide a mapping between the data objects in your software and the data objects defined in the SDK
+Once you have that, you only need to call the import or export service and the rest is taken care of by the SDK.
+
+## Remarks
+Due to how this SDK is defined, with most of the actual implementation details being internal, you should also use one of our bootstrapping packages:
+- **StructuralAnalysisFormat.Bootstrappers.SimpleInjector4**  
+  _If you wish to use SimpleInjector v4_
+- **StructuralAnalysisFormat.Bootstrappers.SimpleInjector5**  
+  _If you wish to use SimpleInjector v5_
```

Comment:
Add links here to the corresponding pages

Reply:
v4: https://docs.simpleinjector.org/en/4.0/
v5: https://docs.simpleinjector.org/en/5.0/